internal/biz/sync_kv_pair.go

```diff
@@ -21,6 +21,8 @@ type KvPair struct {
 	UpdatedExtensionPairs []ExtensionPair
 	SubKeyPairs           []SubKeyPair
 	UpdatedSubKeyPairs    []SubKeyPair
+	SocialKeyPairs        []SocialKvPair
+	UpdatedSocialKeyPairs []SocialKvPair
 }
```

Comment:
Please remove key to keep simle. `SocialKeyPairs` -> `SocialPairs` and `UpdatedSocialKeyPairs` -> `UpdatedSocialPairs`